astropy/units/core.py

```diff
@@ -1211,8 +1195,9 @@ def to(self, other, value=UNITY, equivalencies=[]):
             return self.get_converter(Unit(other), equivalencies)(value)
 
     def in_units(self, other, value=1.0, equivalencies=[]):
-        """
-        Alias for `to` for backward compatibility with pynbody.
+        """Return the converted values in the specified unit.
```

Comment:
I'd prefer something more active like you have elsewhere, say "Convert the value to the specified unit." or "Convert the value from the current unit to the specified one."

If we go this route (fine not to!), then also should adjust `to()`.

Reply:
Do we need both `to()` and `in_units()`? Maybe we should deprecate one of them.

Reply:
Agreed. It's been many years since this forked from pynbody. I'd be in favor of deprecating the method.

Reply:
Yes, agreed, there's no need anymore for that backwards compatibility.

Reply:
I'll open a separate pull request for deprecating this function. And because we are deprecating it then there's no need to update its docstring.